uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_request.scss

```diff
@@ -34,30 +39,57 @@
 
 .request-data {
 
-  dl {
-    @include margin(0);
-  }
+  &-key {
 
-  dt {
-    color: $gray-darker;
-    float: left;
-    font-weight: 600;
-    width: 150px;
+    div,
+    span {
+      color: $gray-darker;
+      font-weight: 600;
+    }
+
+    // Mobile only: remove vertical margin between the key and the value
+    @media only screen and (max-width: #{$global-breakpoint}) {
+      padding-bottom: 0 !important;
+    }
   }
 
-  dd {
-    @include margin(0, left);
+  .row {
 
-    &:not(:last-child) {
-      @include margin(10px, bottom);
+    // Mobile / Desktop: Remove bottom margin on the last row
+    &:last-child {
```

Comment:
i prefer not to make any exceptions and always nest, it's just for consistency's sake (otherwise you get into discussions on when you should nest and when you should not)